recipes-soletta/soletta/soletta_git.bb

```diff
@@ -29,6 +29,9 @@ PACKAGES = " \
          ${PN} \
 "
 
+PACKAGECONFIG ??= "x11"
+PACKAGECONFIG[x11] = ",,cairo atk gtk+3 gdk-pixbuf pango,"
```

Comment:
Patch created here -> https://github.com/solettaproject/meta-soletta/pull/80

Could you guys review it ?